### PR TITLE
feat: main design update

### PR DESCRIPTION
**Pull request by feedm3**
This PR includes the new design.

> Only open to not forget about it. Final merge date and reviews are not yet decided. Some todos are still open.

### CODE REVIEW

**Comment by ghost** [comment]
## :white_check_mark: satellytes.com-main deploy preview ready
* [Deploy preview](https://satellytescommain21751-featmaindesignupdate.gatsbyjs.io)
* [Build logs](https://gatsbyjs.com/dashboard/f215130d-15e3-4176-9252-e13166fe48de/sites/674be368-adf0-4ac5-8fae-f9a4493adc51/builds/16b82b83-6248-4ed2-9e07-823d3c83fb1c/details) · **2m** build time
* Performance **54** :orange_circle: · Accessibility **94** :green_heart: · Best Practices **100** :green_heart: · SEO **86** :orange_circle: — [View full report](https://build-16b82b83-6248-4ed2-9e07-823d3c83fb1c.gtsb.io/reports/lighthouse/index.html)

**Review by georgiee — CHANGES_REQUESTED** [review]
Thanks for all the changes and the hard work 💪 This is a blank change request to hold back the merge until the approved content-wise + quick review of the code changes 🙏

File: crowdin.yml
Comment (georgiee) [inline crowdin.yml:3]: What's that. Do we use crowdin?

File: data/pages/imprint.md
Comment (georgiee) [inline data/pages/imprint.md:1]: - [x] Remove imprint (EN) md

File: data/pages/data-privacy-de.md
Comment (georgiee) [inline data/pages/data-privacy-de.md:1]: This is obsolete and covered through contentful, page dataset `data-privacy`

- [x] Remove data-privacy (DE) md

File: data/pages/data-privacy.md
Comment (georgiee) [inline data/pages/data-privacy.md:1]: - [x] Remove data-privacy (EN) md

File: data/pages/imprint-de.md
Comment (georgiee) [inline data/pages/imprint-de.md:1]: - [x] Remove imprint (DE) md

File: src/assets/customer-logos/01-allianz.svg
Comment (georgiee) [inline src/assets/customer-logos/01-allianz.svg:1]: Those files are still required as they are not reflected through contentful. Fine for me for now

File: src/assets/locales/de/translations.json
Comment (georgiee) [inline src/assets/locales/de/translations.json:1]: much more lean after the transition to contentful. Thanks.

File: src/pages/services.tsx
Comment (georgiee) [inline src/pages/services.tsx:86]: That approach to fetch individual content models of a specific type is pragmatic. I see where you're coming from but there is plenty of space to make this better (we as a team 🙏) once we finally landed this ugly big PR

File: src/pages/index.tsx
Comment (georgiee) [inline src/pages/index.tsx:62]: at least we have the presentational separation present not to mix data and the actual display of that data. Things would look much worse with the many data involved now.

File: src/templates/blog-post-overview.tsx
Comment (georgiee) [inline src/templates/blog-post-overview.tsx:100]: @brataaaan 

I don't understand why you chose to author the content model like this: A section header mode + a separate section paragraph. Because we have theoretically multiple paragraphs? It would be much simpler to have the paragraph integrated in the section header model and saves a content model.

A similar thing happened with the leadbox content model (leadbox + link + contact)

- [x] Please explain the content model. Thank you.

CleanShot 2023-08-29 at 09 42 29

**Review by georgiee — CHANGES_REQUESTED** [review]
Thanks to the team to deliver the design update and the contentful integration. This PR got way too big. We the leadership are mostly to blame as we stretched working on the website update across weeks and months. I feel sorry for that. So please take the following feedback as neutral as possible, so we can improve the website.

Regarding the latest additions it would have been great if we would have some summaries for the two parts in the PR. What happened on the design side (months ago) and what did  @brataaaan deliver in the latest iteration. I got some feedback in the chat about that but a PR of any sizes deserves its own share of communication. Let's keep that in mind for the next time 🙏

I see several issues piling up in this PR:
1. The PR is too big. Way too big but splitting it now is out of question due to the nature of changes. It would take a lot of efforts that isn't worth the result. But the next time we continue working on an already big PR if suggest to deliver a stacked PR on top of the parent branch/PR. That would have made things much easier.

2. Our website code got more complex with the contentful integration and the chosen content model hierarchy. You can see that especially on the page level where the necessary data is fetched and aggregated. I already commented on that: It's pragmatic and it works so we will to do it as proposed, but that's something very high on the technical debt list to keep our website managebable.

3. Speaking of manageable. If I look at our website and then check the implementation I see a beast of code vs a pretty common website. I wonder if I see it like that because I did not check the code for a while combined with the big PR or if we lack someone who really owns the website an feels the urge to refactor.



Regarding my actual review:
I checked out the code but mostly focused on glimpsing through the files in the github file browser. I did not check details. I commented on a few occasions. I checked the deployment too. There is some content to be updated but that's now possible with contentful so it's not blocking this PR.

I've marked some files to remove (md files). I also added some comments that I would like to get acknowledged before merging. So I keep this PR on hold until both parts are resolved.

Thank you very much. I'm so happy once we have cleared this PR. Then we should iterate in some smaller steps.

File: src/templates/blog-post-overview.tsx
Comment (brataaaan) [reply src/templates/blog-post-overview.tsx:100]: Content model Leadbox I can merge (then the type is not so tidy, but we save 2 content types).

For the Section Header React Component, there can be multiple paragraphs, which is why I decided to use another model.

File: crowdin.yml
Comment (georgiee) [reply crowdin.yml:3]: apparently I created that file. Together with @hofmann-felix we integrated that translation manager to have a single source of truth of our texts. We pivoted to contentful mostly so I think we don't need an external system for the few ui translations remaining. We can keep that file for now but it's due for deletion. Maybe Felix can provide some more context if it's still useful or not.

**Review by georgiee — APPROVED** [review]

**Comment by georgiee** [comment]
I will merge to get this big PR out of our way. Thanks everyone for the hard work. Let's get the small things straight so we can enjoy the updated website & content.

File: crowdin.yml
Comment (hofmann-felix) [reply crowdin.yml:3]: Since most of the text is in Contentful, I would remove Crowdin as well. I don't think we will change the remaining texts too often in the future.

**Pull request merged**
